diff --git a/java/com/android/incallui/call/CallRecorder.java b/java/com/android/incallui/call/CallRecorder.java
index 867d5a5..78d028a 100644
--- a/java/com/android/incallui/call/CallRecorder.java
+++ b/java/com/android/incallui/call/CallRecorder.java
@@ -98,14 +98,7 @@ public class CallRecorder implements CallList.Listener {
       if (!isEnabled()) {
           return false;
       }
-      if (RECORD_ALLOWED_STATE_BY_COUNTRY.isEmpty()) {
-          loadAllowedStates();
-      }
-
-      String currentCountryIso = GeoUtil.getCurrentCountryIso(context);
-      Boolean allowedState = RECORD_ALLOWED_STATE_BY_COUNTRY.get(currentCountryIso);
-
-      return allowedState != null && allowedState;
+      return true;
   }
 
   private CallRecorder() {
